test/core/integration/concrete/lockup-linear/streamed-amount-of/streamedAmountOf.tree

```diff
@@ -1,16 +1,17 @@
 StreamedAmountOf_Lockup_Linear_Integration_Concrete_Test
-├── given PENDING status
```

Comment:
Rationale behind removing these tests?

Reply:
duplicated tests (in the common `lockup-base` file):

https://github.com/sablier-labs/v2-core/blob/5764dd34aa56f544ff1e5caaa0be00bf9f9a610e/test/core/integration/concrete/lockup-base/streamed-amount-of/streamedAmountOf.tree#L10-L17

Reply:
[This is](https://github.com/sablier-labs/v2-core/pull/1075#discussion_r1843999835) my reply to this as well.